docs-v2/_docs/extending-wiremock.md

```diff
@@ -165,6 +165,16 @@ stubFor(get(urlEqualTo("/transform")).willReturn(
                 .withTransformer("body-transformer", "newValue", 66)));
 ```
 
+Another convenience provided by the Java API is ability to add transformers instances, without prior registering them as an extension: 
+
+```java
+new WireMockServer(wireMockConfig()); //.extensions(not needed)
+
+stubFor(get(urlEqualTo("/local-transform")).willReturn(aResponse()
+        .withTransformers(new MyTransformer(), new MyOtherTransformer()));
+```
+Transformers added as instances to stubs will be applied locally. The result of `.applyGlobally()` and `.getName()` methods of such transformers will have no effect.
```

Comment:
Since we don't really care about the `.getName()` and `.applyGlobally()` of the transformers added as instances, maybe it would be a good idea to extract a `Transformer` interface detached from the `AbstractTransformer` and `Extension`?
The difficulty here is that the `ResponseTransformer` and `ResponseDefinitionTransformer` would also have to be detached from the current `AbstractTransformer` which might break existing code with `@Override` on `.getName()`. Leaving the worthness of the effort for others to evaluate.